tests/unit/test_extensions/test_litestar/test_dto_integration.py

```diff
@@ -147,6 +152,35 @@ def get_handler() -> Book:
         assert response_callback.json() == json_data
 
 
+class ConcreteBase(Base):
+    pass
+
+
+func_result_query = select(func.count(1)).scalar_subquery()
+model_with_func_query = select(ConcreteBase, func_result_query.label("func_result")).subquery()
+
+
+class ModelWithFunc(Base):
+    __table__ = model_with_func_query
+    func_result: Mapped[int | None] = column_property(model_with_func_query.c.func_result)
```

Comment:
```suggestion
    func_result: Mapped[Optional[int]] = column_property(model_with_func_query.c.func_result)
```
I don't think 3.8 likes the `|` syntax here.